### PR TITLE
Assign correct CSS class to notification widget

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-website/issues/307.

### CODE REVIEW

File: app/javascript/components/notifications/Widget.tsx [hunk 1]
```diff
@@ -20,7 +20,18 @@ function reducer(state: WidgetProps, action: WidgetAction) {
...
+    switch (true) {
+      case state.count === 0:
+        return '--none'
+      case state.count >= 1 && state.count <= 9:
+        return '--digital'
+      case state.count >= 10 && state.count <= 99:
+        return '--double-digit'
+      case state.count >= 100:
+        return '--triple-digit'
+    }
```
Comment (ErikSchierboom) [inline app/javascript/components/notifications/Widget.tsx:33]: I've never seen this pattern before! Very interesting. What made you use this over regular if/else if statements?

File: test/javascript/components/notifications/Widget.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  component = render(<Widget count={10} />)
+  expect(component.container.firstChild).toHaveClass('--double-digit')
+
+  component = render(<Widget count={99} />)
+  expect(component.container.firstChild).toHaveClass('--double-digit')
+
+  component = render(<Widget count={100} />)
+  expect(component.container.firstChild).toHaveClass('--triple-digit')
+})
```
Comment (ErikSchierboom) [inline test/javascript/components/notifications/Widget.test.js:26]: Maybe it's me, but I think I would prefer to have separate tests for the different cases.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
Gonna merge this into my PR. Thanks @kntsoriano! I'll leave @ErikSchierboom's comments to be discussed separately :)

File: app/javascript/components/notifications/Widget.tsx (hunk 1 above)
Comment (SleeplessByte) [reply app/javascript/components/notifications/Widget.tsx:33]: You can use Math too if you'd like:

```typescript
const CLASSES = ['--none', '--digital', '--double-digit', '--triple-digit']

...

const digits = Math.log(state.count) * Math.LOG10E + 1 | 0;
return CLASSES[digits]  || CLASSES[CLASSES.length - 1]
```

File: test/javascript/components/notifications/Widget.test.js (hunk 2 above)
Comment (SleeplessByte) [reply test/javascript/components/notifications/Widget.test.js:26]: It should have separate tests for separate cases.

File: app/javascript/components/notifications/Widget.tsx (hunk 1 above)
Comment (iHiD) [reply app/javascript/components/notifications/Widget.tsx:33]: Please add a comment if you do this!

Comment (SleeplessByte) [reply app/javascript/components/notifications/Widget.tsx:33]: Hah yes!

Comment (kntsoriano) [reply app/javascript/components/notifications/Widget.tsx:33]: @ErikSchierboom I just like how every case is laid out nicely. But I don't have any preference if we go back to use if / else statements.

File: test/javascript/components/notifications/Widget.test.js (hunk 2 above)
Comment (kntsoriano) [reply test/javascript/components/notifications/Widget.test.js:26]: I will separate this to different test cases.

**Pull request merged**